goat/dataset_demonstration.py

```diff
@@ -0,0 +1,34 @@
+import sys
+
+import pandas as pd
+from dataset_utils import remove_unnecessary_symbols
+from spider_utils import TaskType
+
+file_path = f"./{sys.argv[1]}"
```

Comment:
You can use [argparse](https://docs.python.org/3/library/argparse.html) lib for parsing input arguments.

If you are writing a script in python, it is better to put its logic in a separate method and call it inside the if-name-main construct:
```python
if __name__ == "__main__":
    args = ...
    my_script_function(*args)
```

Reply:
Done